scanner/matcher/metadata/migrations/01-init.sql

```diff
@@ -0,0 +1,6 @@
+--- a table to hold the timestamp for the latest vulnerability update.
```

Comment:
Have we considered to store this information in memory but not in DB? Are there any concerns regarding memory consumption or issues that might arise during pod restarts?

Reply:
This is stored in the DB for two main reasons:

1. To share it among all replicas. It's possible different replicas will perform the update process, so each one will need to know the previous timestamp. Storing it in the DB allows us to share it among all replicas.
2. Persistence. This can help us startup faster, as we can read this from the DB to know that we already have vulns, so no need to run the update process immediately nor block scans from succeeding